src/TMedia.h

```diff
@@ -83,15 +83,15 @@ class TMediaData
 private:
     int mMediaProtocol;
     int mMediaType;
-    QString mMediaFileName;
+    QString mMediaFileName {};
     int mMediaVolume;
     int mMediaLoops;
     int mMediaPriority;
     bool mMediaContinue;
-    QString mMediaTag;
-    QString mMediaUrl;
-    QString mMediaKey;
-    QString mMediaAbsolutePathFileName;
+    QString mMediaTag {};
+    QString mMediaUrl {};
+    QString mMediaKey {};
+    QString mMediaAbsolutePathFileName {};
```

Comment:
Give me a green light and I will dig up my Windows machine to test this (as we know my new Macbook is not taking kindly to the latest builds).

Reply:
@mpconley mac builds back online, links are updated.

Reply:
Thanks @vadi2 I'll plan to run some music tests by the end of this weekend.